iOS/samples/sdksample/BingMapsSdkSample/ViewController.swift

```diff
@@ -101,6 +101,7 @@ class ViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSo
                     pushpin.image = self.pinImage
                     pushpin.normalizedAnchorPoint = CGPoint(x: 0.5, y: 1.0)
                 }
+                pushpin.accessibilityLabel = "Pushpin"
```

Comment:
Same suggestion for iOS.  

Also, is there a corresponding change for search?

Reply:
Included search.